plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestPinotIntegrationSmokeTest.java

```diff
@@ -285,7 +285,7 @@ private static GenericRecord createNullRecord()
         Schema schema = getAllTypesAvroSchema();
         // Pinot does not transform the time column value to default null value
         return new GenericRecordBuilder(schema)
-                .set("updated_at", 0L)
+                .set("updated_at", Instant.now().minus(Duration.ofDays(1)).toEpochMilli())
```

Comment:
Document why using something variable rathern than a constant.

Reply:
Some tests depend on the constant value. I'll adjust them and add a comment why.

Reply:
PTAL - added a comment (where the variable is defined). Also had to rewrite some assertions.

Reply:
Constants were used rather than now() so we can have predictable behavior but after thinking about it, it seems that using now() is more predictable, as the constant used in the tests will be older than the retention in a few months:) 

Another alternative is to increase the retention to a something very large and then everything will work.

This looks good to me though, wdyt?

Reply:
I tried disabling the retention but the test setup failed for me. If this looks good let's keep this since this will work always and I think it quite readable?